spec/helpers/produce_items_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
```

Comment:
@tcione I made a helper function to help us format the month list that gets displayed inside the card. I tried to write a test, but I'm stuck. Could you give me some pointers?

I tried to use factories, but it was more confusing. I'm trying to figure out how I could write a produce_item that has many item_seasons.

Reply:
Why stuck? It seems somewhat fine to me. And the usage of factories here is good, I'd do the same, since this grants you control over the data you generate (thus producing predictable results).

That said, I do have a few observations, which I'll comment in place.